beets/util/__init__.py

```diff
@@ -737,10 +737,26 @@ def cpu_count():
         return 1
 
 
+def convert_command_args(args):
+    """Convert command arguments to bytes on Python 2 and strings on
+    Python 3."""
+    def convert(arg):
+        if six.PY2:
+            if isinstance(arg, six.text_type):
+                arg = arg.encode(arg_encoding())
+        else:
+            if isinstance(arg, bytes):
+                arg = arg.decode(arg_encoding(), 'surrogateescape')
+        return arg
+
+    return [convert(a) for a in args]
+
+
```

Comment:
I should mention that I don't have the privileges to do so myself, otherwise I would. Either that I or just can't see any button.